src/common/global.css

```diff
@@ -0,0 +1,75 @@
+/* Text Styles */
+
+@font-face {
+  font-family: 'Quicksand Semibold';
+  src: url('../assets/fonts/Quicksand-SemiBold.ttf');
+}
+
+@font-face {
+  font-family: 'Noto Sans';
+  src: url('../assets/fonts/NotoSans-Regular.ttf');
+}
+
+@font-face {
+  font-family: 'Noto Sans Semibold';
+  src: url('../assets/fonts/NotoSans-SemiBold.ttf');
+}
+
+@font-face {
+  font-family: 'Noto Serif';
+  src: url('../assets/fonts/NotoSerif-Regular.ttf');
+}
+
+/* Header 1 */
+h1 {
+  font-size: 6em;
+  font-family: 'Quicksand Semibold';
+}
+
+/* Header 2 */
+h2 {
+  font-size: 3em;
+  font-family: 'Quicksand Semibold';
+}
+
+/* Header 3 */
+h3 {
+  font-size: 2.25em;
+  font-family: 'Quicksand Semibold';
+}
+
+/* Header 4 */
+h4 {
+  font-size: 1.5em;
+  font-family: 'Quicksand Semibold';
+}
+
+/* Buttons */
+button {
```

Comment:
going to just add all suggestions to a single megacomment here:
- `button`, `input` and `textarea` are all special tags that don't auto apply global styles, but we can do
```JavaScript
input, textarea, button {
    font-family: inherit;
}
```
to get styles into them pretty quickly

- instead of 'main', let's apply the text styles to all elements (`*`) and `body`; `accent` can just be defined as a class that will be imported into all css modules that import this file automatically

- `body` should also have font smoothing (for qol); we can just add the following attributes to implement it:
```CSS
body {
    ... other text styles
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
}
```

- [nice-to-have] global styles is where we should define an app-level site background layer that will occupy at least `min-height: 100%` and `width: 100%` so that other pages don't have to worry about this scaling to make the app fit the entire screen; we can also define default margins and padding here, but these shouldn't be too large (if other components need more padding, they can add them manually through further containers)